PythonScript/src/PythonScript.cpp

```diff
@@ -621,43 +621,37 @@ static void ensurePathExists(const tstring& path)
 
 static void newScript()
 {
-	
-	OPENFILENAMEA ofn;
-	memset(&ofn, 0, sizeof(OPENFILENAMEA));
+	OPENFILENAMEW ofn;

```

Comment:
Should be possible to use the version OPENFILENAME, GetSaveFileName, CreateFile without the W extension as we are building with Unicode and in this case it is anyhow mapped to the W versions, see e.g. docu of https://msdn.microsoft.com/en-us/library/windows/desktop/ms646839(v=vs.85).aspx
at the end of the page in requirements